WebRoot/js/ajax/dwt/widgets/DwtMenu.js

```diff
@@ -1156,7 +1156,7 @@ function(x, y, kbGenerated) {
 		DwtMenu._activeMenuUp = true;
 	}
 
-	this.parent && this.parent.addClassName && this.parent.addClassName(DwtControl.SUB_MENU_ACTIVE);
+	this.parent && this.parent instanceof DwtButton && this.parent.addClassName && this.parent.addClassName(DwtControl.SUB_MENU_ACTIVE);
```

Comment:
How about we add condition to check if parent is not shell? We don't really need to style DwtComboBox at this moment but doesn't harm to apply this class to it. Thoughts? @kalpeshhpatel  @SatyamKumarOnline  @silentsakky